sandbox

```diff
@@ -258,7 +258,8 @@ sandbox () {
     statusline "algod version"
     dc exec algod goal -v
     statusline "Indexer version"
-    INDEXER_VERSION=$(dc exec indexer cmd/algorand-indexer/algorand-indexer -v) && echo ${INDEXER_VERSION} || curl -s "localhost:8980/health?pretty"
+    # the algorand-indexer command doesn't have a version command without either import or daemon arguments
+    INDEXER_VERSION=$(dc exec indexer cmd/algorand-indexer/algorand-indexer import -v) && echo ${INDEXER_VERSION} || curl -s "localhost:8980/health?pretty"
```

Comment:
Import is deprecated, so daemon is a better choice here. Could you also modify the line you found to make version globally available?
```suggestion
    INDEXER_VERSION=$(dc exec indexer cmd/algorand-indexer/algorand-indexer daemon -v) && echo ${INDEXER_VERSION} || curl -s "localhost:8980/health?pretty"
```